packages/space-opera/src/components/camera_settings/components/limits_base.ts

```diff
@@ -134,8 +134,6 @@ export abstract class LimitsBase extends ConnectedLitElement {
     return html`
     <me-section-row label="Minimum">
       <div class="LabelRowContent">
-        <mwc-button id="clear-min-button" unelevated icon="cancel" @click="${
```

Comment:
I don't think these are necessary for this section, and the section does look cleaner without them

Reply:
The image on your other comment was very helpful; would you mind putting a few more of those in so I know what you're talking about?

Reply:
Wait, I think I understand; so now once you set a minimum you can't unset it anymore? That seems not right...

Reply:
this removes the X next to the setting buttons: https://screenshot.googleplex.com/A6yGzfRGLdUkLHJ.png
users can still adjust the slider, so they can set a minimum and maximum just fine